doc/cookbook/source/examples/multiclass_classifier/cartree.rst

```diff
@@ -0,0 +1,47 @@
+==================================
+Classification And Regression Tree
+==================================
+
+Decision tree learning uses a decision tree as a predictive model which maps observations about an item to conclusions about the item's target value.
```

Comment:
Decition tree learning is based on trees as predictive models. (Remove the second sentence)